script/job_generator/versions.py

```diff
@@ -32,7 +32,7 @@
         "11.8",
         "12.0",
     ],
-    HIPCC: ["5.0", "5.1", "5.2", "5.3"],
+    HIPCC: ["5.0.1", "5.1.3", "5.2.3", "5.3.3", "5.4.2"],
```

Comment:
I think 5.4.3 was also released this week.

Reply:
Yes, but there are no docker images (yet) :-(

https://hub.docker.com/r/rocm/dev-ubuntu-20.04/tags?page=1&name=5.4

Reply:
HIP patch levels are not supported. I only care about the major and minor level. So, there are only docker container tags available in the shape of `alpaka-ci-ubuntu20.04-rocm5.3:<container_version>`.

If we would support different patch level for rocm, we need to store different versions of the container, like:

* alpaka-ci-ubuntu20.04-rocm5.3.0:3.0
* alpaka-ci-ubuntu20.04-rocm5.3.1:3.0
* alpaka-ci-ubuntu20.04-rocm5.3.2:3.0
* alpaka-ci-ubuntu20.04-rocm5.3.3:3.0

Or we ignore the caching mechanism and install hip in each test job like before.

I would avoid handling the patch levels until it causes problems. At the moment, it would be only additional work without benefit.

Reply:
The automatically pull the latest patch level, when they are built. Means, it depends a little bit, when the last push on the master branch happened.